tcrconvert/convert.py

```diff
@@ -33,19 +35,27 @@ def choose_lookup(frm, to, species='human'):
     PosixPath('/home/emmabishop/anaconda3/envs/tcrconvert/lib/python3.12/site-packages/tcrconvert/data/human/lookup.csv')
```

Comment:
note that for `doctest` paths will vary based on the machine/server they're running on, so https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS this may help with that

Reply:
True, the paths with vary. I didn't know about the ELLIPSIS option so thanks for showing me! Switched to using `...` in relevant paths in 23051cb82c64c1a127cfc94f664df34bb28583f1.